### ISSUE TITLE
need documentation on batch mode outputs

### ISSUE THREAD

**Issue by stephenturner** [pachterlab/kallisto#113]
what's in the matrix.ec, matrix.tsv, and matrix.cells, and how to join those up together to create a tcc matrix of sample (col) by tcc (row)?

**Comment by flying-sheep** [comment]
yes! and also there needs to be a way to use batch mode without UMIs.
